### ISSUE TITLE
Calendar Basic Outline

### ISSUE THREAD

**Issue by Zeyddd** [CSE-110-Group-13/warm-up#4]
I've created a basic outline for the calendar so far. I'm missing the days which I can't seem to get working for some reason, and the design so far sucks. It needs accessibility with the keyboard, and if JS doesn't run on a device, this whole thing won't work, so we either commit to JS only or have an HTML fallback option. I talked to the prof and he says it's possible to have an HTML fallback by doing some server-side coding, which I don't really understand so I'll leave that for you guys.
